src/serializer.ts

```diff
@@ -44,16 +47,29 @@ class Serializer {
       Fee: '',
     }
 
-    object.Sequence = transaction.getSequence()
-    object.SigningPubKey = Utils.toHex(transaction.getSigningPublicKey_asU8())
-    object.LastLedgerSequence = transaction.getLastLedgerSequence()
+    const sequence = transaction.getSequence()?.getValue()
+    if (sequence) {
+      object.Sequence = sequence
+    }
+
+    const signingPubKeyBytes = transaction
+      .getSigningPublicKey()
+      ?.getValue_asU8()
+    if (signingPubKeyBytes) {
+      object.SigningPubKey = Utils.toHex(signingPubKeyBytes)
+    }
+
+    const lastLedgerSequence = transaction.getLastLedgerSequence()?.getValue()
+    if (lastLedgerSequence) {
+      object.LastLedgerSequence = lastLedgerSequence
+    }
 
     // Convert account field, handling X-Addresses if needed.
     const accountAddress = transaction.getAccount()
     if (!accountAddress) {
       return
     }
-    const account = accountAddress.getAddress()
+    const account = accountAddress.getValue()?.getAddress()
```

Comment:
What's with the `.getValue()` everywhere? Is that part of the new Protobuf?

Reply:
Yeah, there's a new `value` wrapper on a bunch of common types that added a layer of abstraction which was useful in the network handling layer.